docs/publish/centralized-deployment.md

```diff
@@ -45,38 +45,37 @@ If your organization meets all requirements, complete the following steps to pub
 
 1. Sign in to Office 365 with your work or school account.
 2. Select the app launcher icon in the upper-left and choose **Admin**.
-3. In the navigation menu, choose **Settings** > **Services & add-ins**.
+3. In the navigation menu, press **Show more**, then choose **Settings** > **Services & add-ins**.
 4. If you see a message on the top of the page announcing the new Office 365 admin center, choose the message to go to the Admin Center Preview (see [About the Office 365 admin center](https://support.office.com/en-ie/article/About-the-Office-365-admin-center-758befc4-0888-4009-9f14-0d147402fd23)).
-5. Choose **Upload Add-in** at the top of the page. 
-6. Choose one of the following options on the **Centralized Deployment** page:
+5. Choose **Deploy Add-In** at the top of the page. 
+6. Choose *Next** after reviewing the requirements.
+7. Choose one of the following options on the **Centralized Deployment** page:
 
-    - **I want to add an add-in from AppSource.**
+    - **I want to add an Add-In from the Office Store.**
```

Comment:
The "Office Store" was officially rebranded as "AppSource" several months ago (and our doc set was updated to reflect this change in name at that time) -- so I'm surprised to hear that the UI for Centralized Deployment hasn't also been rebranded accordingly.  If you're still seeing "Office Store" in the CD UI, we probably should escalate that to the PM who owns Centralized Deployment to see if/when they're planning to change "Office Store" to "AppSource".  I'm not sure who the PM is though...maybe ask Reeza or Sean Laberee?

Reply:
I believe Alan Mulhall is the PM owning CD. I've sent mail regarding this to him. Should our walkthrough reflect the current state or the proper branding?

Reply:
IMO, our docs should reflect current state, since that's the best user experience -- i.e., the docs matching what the user is seeing in the UI.  

However, presumably the CD team is planning to update the UI sometime in the near future to reflect the rebranding of "Office Store" to "AppSource" -- so, at the very least, we should try to identify when that's going to happen and log/assign a work item for our team in VSO with that info so that we can be sure to update this article when the UI changes.